app/controllers/TestUserController.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+class UsersController extends \BaseController {
+
+	/**
+	 * Display a listing of the resource.
+	 * GET /users
+	 *
+	 * @return Response
+	 */
+	public function index()
+	{		
+		$user 		= 	Auth::user();
+		$created 	= 	$user->tasks_created;
+		$completed 	= 	$user->tasks_completed;
+
+		$pTitle		=	Auth::user()->full_name;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused "$pTitle" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.109:9000/coding_rules#rule_key=php%3AS1481)